src/elements/paginator/compact-paginator/compact-paginator.scss

```diff
@@ -0,0 +1,33 @@
+@use '../../core/styles/index' as sbb;
```

Comment:
For pager position = end, shouldn't it be right-aligned like the normal paginator? maybe a question for @mcilurzo

Reply:
Manuel has the last word but I assume yes